ui/app/styles/core/buttons.scss

```diff
@@ -183,6 +183,10 @@ $button-box-shadow-standard: 0 3px 1px 0 rgba($black, 0.12);
     font-weight: $font-weight-semibold;
   }
 
+  &.has-text-danger {
```

Comment:
effects one other button that uses has-text-danger. confirmed with design this fix for both buttons is desired (The MFA designs had the border).